ext/xmlreader/tests/XML_error.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+XMLReader: XML Error
+--SKIPIF--
+<?php if (!extension_loaded("xmlreader")) print "skip XMLReader extension required"; ?>
+--FILE--
+<?php
+
+$reader = new XMLReader();
+var_dump($reader->XML());
+?>
+--EXPECTF--
+Warning: XMLReader::XML() expects at least 1 parameter, 0 given in %s on line %d
```

Comment:
Please, remove these kind of tests, we don’t test ZPP failures. 

For the rest, :+1: for me :)